### PR TITLE
GUACAMOLE-423: Bump version numbers in manual to 0.9.14.

### PR DESCRIPTION
**Pull request by mike-jumper**

### CODE REVIEW

File: src/chapters/jdbc-auth.xml [hunk 1]
```diff
@@ -227,20 +227,13 @@ Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
...
-<prompt>$</prompt></screen>
-                </informalexample>
+                    version of Guacamole older than <replaceable>VERSION</replaceable>.</para>
                 <para>These scripts are incremental and, when relevant, <emphasis>must be run in
                         order</emphasis>. For example, if you are upgrading an existing database
-                    from version 0.9.10-incubating, you would need to run the
-                        <filename>upgrade-pre-0.9.11.sql</filename> script (because 0.9.10 is older
-                    than 0.9.11), followed by the <filename>upgrade-pre-0.9.13.sql</filename> script
-                    (because 0.9.10 is also older than 0.9.13):</para>
+                    from version 0.9.10-incubating to version 0.9.13-incubating, you would need to
```
Comment (necouchman) [inline src/chapters/jdbc-auth.xml]: Was there a reason to leave the -incubating reference in here and not make the example for 0.9.14, without that?  I think it's good either way, just want to make sure.

Comment (mike-jumper) [reply src/chapters/jdbc-auth.xml]: My only reason was to avoid having to rewrite the example.

Comment (necouchman) [reply src/chapters/jdbc-auth.xml]: That's fair.  Like I said, fine with me, just wanted to make sure.

Comment (mike-jumper) [reply src/chapters/jdbc-auth.xml]: (...with the rest of the rewording here to avoid having to rewrite the example for future releases, as well)

**Pull request merged**